app/(tabs)/_layout.tsx

```diff
@@ -7,7 +7,7 @@ import { useColorScheme } from '@/hooks/useColorScheme';
 
 export default function TabLayout() {
   const colorScheme = useColorScheme();
-
+// TODO: INSTEAD OF TOASTS, USE THE NAVIGATION STACK 
```

Comment:
Why is this comment here ?
This kind of comments belong more in the scrum board (it's an issue)